kubernetes/helm/nginx-ingress/values.yaml

```diff
@@ -1,6 +1,8 @@
 controller:
   autoscaling:
     enabled: true
+    minReplicas: 3
+    maxReplicas: 18
```

Comment:
This is just for the autoscaling controller, but doesn't safely guarantee that there's enough pods during a deployment.  Set also controller.replicaCount and controller.minAvailable.